Realm/Realm.Weaver/Analytics/AnalyticsUtils.cs

```diff
@@ -136,34 +137,39 @@ public static (string Name, string Version) GetFrameworkAndVersion(ModuleDefinit
             }
         }
 
-        public static string GetLanguageVersion(string targetFramework)
+        public static string GetLanguageVersion(string netFramework, string netFrameworkVersion)
         {
             // We don't have a reliable way to get the version in the weaver so we're using the default version
             // associated with the framework.
             // Values taken from https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/configure-language-version
-            if (targetFramework.ContainsIgnoreCase("netcoreapp2") ||
-                targetFramework.ContainsIgnoreCase("netframework"))
+            if (!netFramework.ContainsIgnoreCase("net"))
+            {
+                return Unknown($"Likely not the model assembly, but the platform specific one: {netFramework}");
```

Comment:
This is going to be a very long string to send. Should we move this to a comment and instead use `Unknown(netFramework)`?